src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const cssPropertiesDoubleDot = cssPropertiesDotComa
+    .map((item) => item.split(':'))
+    .map((splittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      splittedCssProperty.map((propertyPart) => propertyPart.trim()));
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability.

Good Example:
const cssPropertiesDoubleDot = cssPropertiesDotComa
  .map((item) => item.split(':'))
  .map((splittedCssProperty) => splittedCssProperty.map((propertyPart) => propertyPart.trim()));